docs/pbc/all/dynamic-multistore/202404.0/base-shop/dynamic-multistore-feature-overview.md

```diff
@@ -80,3 +80,84 @@ Normal storefront APIs can be used to fetch data from stores using the Store HTT
 
 ![storefront-api](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/dynamic-multistore/dynamic-multistore.md/storefront-api.png)
+
+## Adding of a new store via data import
+
+Example for DE store configuration:
+
+**data/import/common/{REGION}/store.csv**
+
+```csv
+name
+DE
+AT
+```
+
+| Column     | REQUIRED | Data Type | Data Example | Data Explanation |
+|------------| --- | --- | --- | --- |
+|name        |mandatory |string | DE | Define store name. |
+
+Configure a new store as described here: [Import configuration for store](/docs/pbc/all/dynamic-multistore/202311.0/base-shop/install-and-upgrade/install-features/install-dynamic-multistore.html#import-data)
+
+
+Adding a new store via the Back Office is easier and faster, but in this case, you should add this store across all environments.
+Data import allows you to add and configure a new store once and deploy it across all environments.
+
+Note that when you add a new store you should also assign all store-related entities to it for enabling them for customers:
+- Products
+- Categories
+- CMS entities
+- Prices
+- etc.
+
+To avoid manual assignment of entities in Back Office to the store you can use the data import for store-related entities as well.
+
+See: [Import minimum set of data for store](/docs/pbc/all/dynamic-multistore/202404.0/base-shop/import-stores.html)
+
+
+## What is changed when DMS is enabled?
+
+DMS enables the following changes to the project:
+
+In represented examples below there are two regions: EU and US.
+
+EU region has two stores: DE and AT. US region has one store: US.
+
+- Urls for Yves, Back Office, Merchant Portal and Glue contain region instead of store name.
+  
+  Example for local environment: https://backoffice.eu.mysprykershop.com instead of https://backoffice.de.mysprykershop.com 
+  ![local-dashboard-non-dms]()
+  ![local-dashboard-dms]() 
+
+- RabbitMQ virtual hosts contain region instead of store.
+  
+  For example eu-docker instead of de-docker.
+  ![rabbitmq-virtual-hosts-non-dms]()
+  ![rabbitmq-virtual-hosts-dms]()
+
+- Jenkins job names contain region instead of store. 
+
+  For example EU_queue-worker-start instead of DE_queue-worker-start.
+  ![jenkins-jobs-non-dms]()
+  ![jenkins-jobs-dms](
+
+- Elasticsearch indexes contain store as a part of the index name for DMS enabled and disabled modes.
+  ![elasticsearch-indexes-non-dms]()
+  ![elasticsearch-indexes-dms]()
+
+- Redis keys contain store as a part of the key name for DMS enabled and disabled modes.
+  ![redis-keys-non-dms]()
+  ![redis-keys-dms]()
+
+- When DMS is enabled customer can switch between available stores for a region.
+
+  When customer changes a store it's set to a query param `_store`.
+
+  Store is read from query param `_store` and set to session under the key `current_store`. It's used for fetching store-related data.
+  ![yves-store-switcher]()
```

Comment:
<img width="1264" alt="store-switcher" src="https://github.com/user-attachments/assets/c4a6238b-6f52-4962-968d-927a8bdda504">

Reply:
Please use the homepage from any public demo shop instead of suite-nonsplit

Reply:
<img width="1118" alt="yves-store-switcher" src="https://github.com/user-attachments/assets/003c91a9-a26d-49a7-ba80-700e232885ec">